package 2.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "my-app",
+  "version": "1.0.0",
+  "main": "node_modules/expo/AppEntry.js",
+  "scripts": {
+    "start": "expo start",
+    "android": "expo start --android",
+    "ios": "expo start --ios",
+    "web": "expo start --web",
+    "eject": "expo eject",
+    "lint": "eslint ./"
+  },
+  "dependencies": {
+    "expo": "~43.0.2",
+    "expo-status-bar": "~1.1.0",
+    "firebase": "^9.6.1",
+    "react": "17.0.1",
+    "react-dom": "17.0.1",
+    "react-native": "0.64.3",
+    "react-native-paper": "^4.11.1",
+    "react-native-qrcode-svg": "^6.1.2",
+    "react-native-svg": "^12.1.1",
+    "react-native-vector-icons": "^9.0.0",
+    "react-native-web": "0.17.1",
+    "styled-components": "^5.3.3"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.12.9",
+    "eslint": "^7.32.0",
+    "eslint-config-hackreactor": "github:reactorcore/eslint-config-hackreactor",
+    "eslint-plugin-react": "^7.27.1",
+    "eslint-plugin-react-native": "^4.0.0",
+    "prettier": "^2.5.1"
+  },
+  "private": true
+}
```

Comment:
Why do we need a package2.json file?

Reply:
I was curious myself too. I deleted it and updated it.